setup.py

```diff
@@ -82,10 +82,11 @@ def create_readme_rst():
           "torch >= 0.3.0",
           "unidecode",
           "inflect",
-          "librosa",
+          "librosa == 0.5.1",
           "numba",
           "lws <= 1.0",
           "nltk",
+		  "requests",
```

Comment:
Please consider spaces

Reply:
Another vestigial element :/
will change it